README.md

```diff
@@ -185,6 +185,14 @@ Note that although the annotation containing router configuration for each of th
 | deis-router          | workerProcesses  | `string`   | `auto` (number of CPU cores) | Number of worker processes to start. |
 | deis-router          | workerConnections | `integer` | `768`         | Maximum number of simultaneous connections that can be opened by a worker process. |
 | deis-router          | defaultTimeout   | `integer`  | 1300          | Default timeout value in seconds.  Should be greater than the front-facing load balancer's timeout value. |
+| deis-router          | useGzip          | `boolean`  | `true`        | Whether to use gzip compression. |
```

Comment:
oh I see that now - You make a valid point about the default values there and as such `useGizp` makes sense if I wouldn't want it at all, just from the outside it feels like an extra argument to deal with but its on by default so no big deal. Maybe we can explore different approaches when you are happy with alpha :)

PR is ready either way